storage/src/lib.rs

```diff
@@ -1,12 +1,11 @@
 // Copyright (c) SimpleStaking and Tezedge Contributors
 // SPDX-License-Identifier: MIT
-#![forbid(unsafe_code)]
```

Comment:
Was this because of the `Send` trait? Please leave it commented out (instead of deleted) with a `TODO - TE-261: uncomment once storage has moved to protocol runner`. We really want to forbid any `unsafe` code.

Reply:
Yes it's because of the Send trait, I added a TODO